packages/expect/src/__tests__/__snapshots__/spy_matchers.test.js.snap

```diff
@@ -1059,3 +1266,742 @@ exports[`toHaveBeenNthCalledWith works with trailing undefined arguments 1`] = `
+
+Expected mock function to not have last returned:
+  <green>Set {1, 2}</>
+But it last returned exactly:
+  <red>Set {1, 2}</>"
+`;
+
+exports[`toHaveLastReturnedWith works with Set 2`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).toHaveLastReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function to have last returned:
+  <green>Set {3, 4}</>
+But it last returned:
+  <red>Set {1, 2}</>"
+`;
+
+exports[`toHaveLastReturnedWith works with argument that does match 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).not.toHaveLastReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function to not have last returned:
+  <green>\\"foo\\"</>
+But it last returned exactly:
+  <red>\\"foo\\"</>"
+`;
+
+exports[`toHaveLastReturnedWith works with argument that does not match 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).toHaveLastReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function to have last returned:
+  <green>\\"bar\\"</>
+But it last returned:
+  <red>\\"foo\\"</>"
+`;
+
+exports[`toHaveNthReturnedWith should reject non integer nth value 1`] = `"nth value <red>0.1</> must be a positive integer greater than <green>0</>"`;
+
+exports[`toHaveNthReturnedWith should reject nth value smaller than 1 1`] = `"nth value <red>0</> must be a positive integer greater than <green>0</>"`;
+
+exports[`toHaveNthReturnedWith should replace 1st, 2nd, 3rd with first, second, third 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).toHaveNthReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function first call to have returned with:
+  <green>\\"bar1\\"</>
+But the first call returned with:
+  <red>\\"foo1\\"</>"
+`;
+
+exports[`toHaveNthReturnedWith should replace 1st, 2nd, 3rd with first, second, third 2`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).not.toHaveNthReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function first call to not have returned with:
+  <green>\\"foo1\\"</>
+But the first call returned exactly:
+  <red>\\"foo1\\"</>"
+`;
+
+exports[`toHaveNthReturnedWith works only on spies or jest.fn 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>)[.not].toHaveNthReturnedWith(</><dim>)</>
+
+<red>jest.fn()</> value must be a mock function or spy.
+Received:
+  function: <red>[Function fn]</>"
+`;
+
+exports[`toHaveNthReturnedWith works when not called 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).toHaveNthReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function first call to have returned with:
+  <green>\\"foo\\"</>
+But it did <red>not return</>"
+`;
+
+exports[`toHaveNthReturnedWith works with Immutable.js objects directly created 1`] = `
+"<dim>expect(</><red>jest.fn()</><dim>).not.toHaveNthReturnedWith(</><green>expected</><dim>)</>
+
+Expected mock function first call to not have returned with:
+  <green>Immutable.Map {\\"a\\": {\\"b\\": \\"c\\"}}</>
+But the first call returned exactly:
+  <red>Immutable.Map {\\"a\\": {\\"b\\": \\"c\\"}}</>"
```

Comment:
I'm not sure if we need the same value serialised twice. Visual clutter, no?

Reply:
Was on the fence and choose to include for consistency (all failures print a "but it returned")

Reply:
I agree though, it's silly to print. That's enough of an objection to remove 👍

Reply:
was it ever?